nextcloudappstore/core/models.py

```diff
@@ -254,6 +254,19 @@ def compatible_unstable_releases(self, platform_version, inclusive=True):
             reverse=True,
         )
 
+    def is_outdated(self):
+        """Checks if an app has been released in last 3 recent platform versions
+
+        :return: True if not compatible, otherwise false
+        """
+
+        release_versions = list(self.latest_releases_by_platform_v().keys())
+        if not release_versions:
+            return True
+        max_release_version = Version(pad_max_inc_version(release_versions[0]))
```

Comment:
Unfortunately, I don't think this will work as `release_versions` is not sorted.
Have you tested this?

Reply:
They should be sorted, no? Look in the implementation for `latest_releases_by_platform_v`.

Reply:
Can you point out the place in the implementation where it sorts them?

Reply:
Ah sorry, I must have confused it with another function somewhere along the way. Will fix it now.